paymentsheet/src/main/java/com/stripe/android/paymentsheet/model/SupportedPaymentMethod.kt

```diff
@@ -59,11 +59,34 @@ import kotlinx.parcelize.Parcelize
  * compose model.
  */
 internal sealed class SupportedPaymentMethod(
+    /**
+     * This describes the PaymentMethod Type as described
+     * https://stripe.com/docs/api/payment_intents/create#create_payment_intent-payment_method_types
+     */
     val type: PaymentMethod.Type,
+
+    /** This describes the name that appears under the selector. */
     @StringRes val displayNameResource: Int,
+
+    /** This describes the image in the LPM selector */
```

Comment:
thinking out loud: Would it be valuable to describe where these images should come from?